helmfile/charts/networkpolicy-common/values.yaml

```diff
@@ -0,0 +1,40 @@
+fullnameOverride: ""
+nameOverride: ""
+
+global:
+  apiserver:
```

Comment:
We have changed to scapiserver and wcapiserver. But i would like to go back to just apiserver. So i could move more common np into common.

Reply:
@viktor-f @Xartos :eyes:

Reply:
@robinAwallace Look at the `values/networkpolicies-common.yaml.gotmpl` as well.

Reply:
Ah perfect :+1: